eth2deposit/cli/generate_keys.py

```diff
@@ -36,23 +36,23 @@ def validate_password(cts: click.Context, param: Any, password: str) -> str:
     try:
         validate_password_strength(password)
     except ValidationError as e:
-        click.echo(f'Error: {e} Please retype.')
+        click.echo(f'Error: {e} Please retype.') # Translate
     else:
         is_valid_password = True
 
     while not is_valid_password:
-        password = get_password(text='Type the password that secures your validator keystore(s)')
+        password = get_password(text='Type the password that secures your validator keystore(s)') # Translate
```

Comment:
Let's make it more clear this step creates a password, e.g. "Create a password that will secure your validator keystore(s)"